### DISCUSSION TITLE
Cannot set proxy which needs authentication

### DISCUSSION THREAD

**Discussion by kodbaumjoci** [cypress-io/cypress#27850] [Questions and Help]
Hi Folks, 
I would like to use Cypress on my company PC where I also need to use VPN. I added my http_proxy environment variable on the format which contains my credentials as well "http://myUsername:myPassword@url:port_number" , I can see under settings that cypress recognized it, however I always get an error that says "> Error: Error establishing proxy connection. Response from server was: HTTP/1.1 407 Proxy Authentication Required"
What did I miss? When I just enter the same url what I populated within cy.visit then I can access it without any problem.

**Comment by Jordsterking** [comment]
Same thing happens to me.  It seems to be inconsistent though.  Sometimes cypress runs fine and all tests run perfect but other times the proxy error comes up.  I can see my proxy in the Settings-Device Settings-Proxy Settings but still have 50%. chance of getting proxy error when running the specs.

**Reply by Jordsterking** [reply]
<img width="524" alt="image" src="https://github.com/cypress-io/cypress/assets/16885361/8635fc98-6203-4bed-a860-f7af28492eba">
